connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -259,6 +259,20 @@ public boolean isSinkConnector(String connName) {
      * @return a list of updated tasks properties.
      */
     public List<Map<String, String>> connectorTaskConfigs(String connName, int maxTasks, List<String> sinkTopics) {
+        return connectorTaskConfigs(connName, maxTasks, sinkTopics, null);
+    }
```

Comment:
This signature is no longer invoked anywhere within this project, but I keep it here since it's a public method that other projects could conceivably be invoking.